.github/dependabot.yml

```diff
@@ -9,3 +9,5 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "daily"
+    reviewers:
+      - mark-stopka
```

Comment:
I would prefer to keep CI workflows agnostic of particular maintainers. Makes it harder for forks to sync with upstream.

Reply:
What forks do you mean?